### DISCUSSION TITLE
Blinking caret spacing in TextBox

### DISCUSSION THREAD

**Discussion by JohnKesko** [AvaloniaUI/Avalonia#12481] [Q&A]
Anyone else experiencing this behaviour with the TextBox on macOS? Have not checked on Windows so don't know how it looks there.

Basic styling:
![Caret_style](https://github.com/AvaloniaUI/Avalonia/assets/87022265/32a3f3a3-3252-4c1b-8eec-74d40d98a88d)

Default:
![Caret_nostyle](https://github.com/AvaloniaUI/Avalonia/assets/87022265/c9cafa40-d4c1-4939-b0d4-62d420b72570)

**Comment by Gillibald** [comment]
Is the caret always inside the glyph or just for the `f` character?

**Reply by JohnKesko** [reply]
It's "worst" on `f` and `r` but it affects all of them even though it's very small. I think `P` shows how it looks for them all. You see how it chops off the end a tiny bit.

![Caret_P_nochop](https://github.com/AvaloniaUI/Avalonia/assets/87022265/9c05b1f4-7029-4c4d-a16b-014fb9e543d6)

![Caret_P](https://github.com/AvaloniaUI/Avalonia/assets/87022265/4638cbcb-bda1-4e98-9a82-9ec700b4d0f1)
